src/booleans.js

```diff
@@ -1,61 +1,96 @@
 const negate = a => {
-  // your code here
+  return !a;
 };
 
 const both = (a, b) => {
-  // your code here
+  return a && b;
 };
 
 const either = (a, b) => {
-  // your code here
+  return a || b;
 };
 
 const none = (a, b) => {
-  // your code here
+  return !a && !b;
 };
 
 const one = (a, b) => {
-  // your code here
+  return (a && !b) || (!a && b);
 };
 
 const truthiness = a => {
-  // your code here
+  return a ? true : false;
 };
 
+// Tried an if statement for this, but can't figure out why it won't work:
+
+// const truthiness = a => {
+// if (a == true) {
+//   return true;
+// };
+// return false;
+// };
+
 const isEqual = (a, b) => {
-  // your code here
+  if (a === b) {
+    return true;
+  }
+  return false;
 };
 
 const isGreaterThan = (a, b) => {
-  // your code here
+  if (a > b) {
+    return true;
+  }
+  return false;
 };
 
 const isLessThanOrEqualTo = (a, b) => {
-  // your code here
+  if (a <= b) {
+    return true;
+  }
+  return false;
 };
 
 const isOdd = a => {
-  // your code here
+  if (a % 2 === 0) {
+    return false;
+  }
+  return true;
 };
 
+
 const isEven = a => {
-  // your code here
+  if (a % 2 === 0) {
```

Comment:
Same as before.
If you only have two possibilities in the answers, that's kind of the cleanest way.